src/crkeng/site/settings.py

```diff
@@ -66,3 +66,14 @@
         },
     },
 }
+
+# The order in which paradigm sizes will be presented to the user.
+# The first size in this list is the "default".
+# Make sure to exhaustively specify all size options available!
+MORPHODICT_PARADIGM_SIZES = [
+    # The most user-friendly size should be first:
+    "basic",
+    # Then, a more complete paradigm layout:
+    "full",
+    # Variants for linguists go here:
+]
```

Comment:
@aarppe this is where you would add "algonquianist" paradigm size option (#860).